core/tests/protractor_utils/CreatorDashboardPage.js

```diff
@@ -91,6 +93,14 @@ var CreatorDashboardPage = function() {
     waitFor.pageToFullyLoad();
   };
 
+  this.clickCreateNewExplorationButton = function() {
+    waitFor.elementToBeClickable(
+      createNewExplorationButton,
+      'Create Exploration button on mobile takes too long to be clickable');
```

Comment:
This error message doesn't make sense since the function name has nothing to do with mobile.

Is this the navbar button or the button-in-the-page? Prefer the latter if possible, and remove the reference to mobile if that's the case.

(If navbar then maybe we should make a separate protractor PageObject file for that.)

Reply:
This is the button-in-page. The message has been modified accordingly. Thanks!